ming/metadata.pyi

```diff
@@ -193,16 +193,16 @@ class _ClassManager(Generic[M]):
     def remove(self, spec_or_id: Union[MongoFilter, ObjectId] = None, **kwargs) -> ChangeResult: ...
     def count(self) -> int: ...
     def update_partial(self, filter: MongoFilter, fields: dict, **kwargs) -> ChangeResult: ...
```

Comment:
`ChangeResult` is an alias for `dict` (above).  This should be updated to pymongo UpdateResult and DeleteResult